pkg/operator/controller/controller.go

```diff
@@ -247,6 +249,19 @@ func (r *reconciler) enforceEffectiveHighAvailability(ci *ingressv1alpha1.Cluste
 	return nil
 }
 
+// ensureClusterIngressFinalized adds ClusterIngressFinalizer to ingress
+// if it does not exist and does not contain a DeletionTimestamp.
+func (r *reconciler) ensureClusterIngressFinalized(ingress *ingressv1alpha1.ClusterIngress) error {
```

Comment:
`Finalized` makes it sound like the method finalizes the clusteringress, similar to how `finalizeLoadBalancerService` finalizes the service.  How about `HasFinalizer`?

Reply:
I have updated to `enforceFinalizer`.